applications/asset_tracker_v2/sample.yaml

```diff
@@ -107,7 +113,7 @@ tests:
   applications.asset_tracker_v2.lwm2m.debug-memfault:
     build_only: true
     build_on_all: true
-    platform_allow: nrf9160dk_nrf9160_ns thingy91_nrf9160_ns
+    platform_allow: nrf9160dk_nrf9160_ns
```

Comment:
We are hitting flash limitations with lwm2m-debug-memfault build for thingy:
```
zephyr/zephyr_pre0.elf section `datas' will not fit in region `FLASH'
region `FLASH' overflowed by 3916 bytes
```

Removing thingy from this configuration was my quick change to see if there are other issues.
@simensrostad What should we do?
We are struggling with the same issue in modem shell too and every now and then we need to change configurations to make them fit into flash for thingy. That's not nice and I suggest we do a more radical solution to make sure that there is no configuration which is filling 99% of the flash because that is just waiting for a problem to arise.

Reply:
Try to remove the `overlay-debug.conf` and rename the test case.

Reply:
Done and build is passing now. I'll close this comment.